### PR TITLE
update ember-cli et al

### PR DESCRIPTION
**Pull request by marcoow**
- 🆙 
- remove bower

### CODE REVIEW

**Comment by Turbo87** [comment]
@marcoow CI is failing

**Comment by marcoow** [comment]
@Turbo87 fixed

File: .travis.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ sudo: false
 cache:
   directories:
     - $HOME/.npm
-    - $HOME/.cache # includes bowers cache
+    - $HOME/.cache
```
Comment (Turbo87) [inline .travis.yml]: is this still needed if we don't use bower?

Comment (marcoow) [inline .travis.yml]: Hm, guess not - removed

**Pull request merged**